types/name-initials/index.d.ts

```diff
@@ -0,0 +1,7 @@
+// Type definitions for name-initials 0.1
+// Project: https://github.com/ericvera/name-initials#readme
+// Definitions by: Shirsh Zibbu <https://github.com/zhirzh>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+function nameInitials(name: string): string;
+export = nameInitials;
```

Comment:
Does this still work with ES6 imports?  I would have guessed you needed `--allowSyntheticDefaultImports` or `--esModuleInterop`.